scripts/crowdloan-rewards/common/performVestings.ts

```diff
@@ -0,0 +1,118 @@
+require("dotenv").config();
+const BN = require("bn.js");
+const { ApiPromise, WsProvider, Keyring } = require("@polkadot/api");
+const { encodeAddress, cryptoWaitReady } = require("@polkadot/util-crypto");
+const { stringToU8a } = require("@polkadot/util");
+const assert = require("assert");
+
+import { DynamicVestingInfo } from './generateVestings'
+
+const ACCOUNT_SECRET = process.env.ACCOUNT_SECRET || "//Alice";
+const RPC = process.env.RPC_SERVER || "ws://127.0.0.1:9988";
+
+const hdxAddress = (pubKey) => encodeAddress(pubKey, 63);
+const vestingsAccPubKey = stringToU8a("modlpy/vstng".padEnd(32, "\0"));
+const VESTINGS_ACCOUNT = hdxAddress(vestingsAccPubKey);
+
+export const log = function(text: String) {
+  console.log(text);
+}
+
+const chunkify = (a, size) => Array(Math.ceil(a.length / size)).fill(a).map((_, i) => a.slice(i * size, i * size + size));
```

Comment:
lol, TS takes this?

Reply:
TS will take any JS you throw at it .. you could actually rename a file from .js to .ts and it will work :D

Reply:
but not without complaining usually, but mby it is lining that ruins my day